package.json

```diff
@@ -45,6 +45,7 @@
     "dependencies": {
         "@react-native-community/checkbox": "^0.5.8",
         "@react-native-picker/picker": "^2.1.0",
+        "@react-navigation/native": "^6.0.8",
```

Comment:
let's put this as a peer dependency, and restrict this to indicate that only can use any 6 version